reversion_compare/templates/reversion-compare/compare.html

```diff
@@ -6,15 +6,25 @@
     /* minimal style for the diffs */
     pre.highlight {
         max-width: 900px;
-        white-space: pre-line;
```

Comment:
@jedie Quick heads up, I just upgraded to 0.13.1 and noticed that this change breaks line wrapping. Here's what it's supposed to look like (with `white-space: pre-line`):

![Screen Shot 2021-02-09 at 9 37 45 AM](https://user-images.githubusercontent.com/306708/107403937-8f906b80-6aba-11eb-806a-3e08066756a0.png)

Here's what it currently looks like in 0.13.1 (no `white-space: pre-line`):

![Screen Shot 2021-02-09 at 9 37 36 AM](https://user-images.githubusercontent.com/306708/107404042-af279400-6aba-11eb-91c8-7e24b794f983.png)

Without `white-space: pre-line`, long lines extend past the max-width and the diff looks broken.

There was probably another reason why you removed the `white-space` setting? Or was it accidental? Let me know and I can put together a quick PR to add it back. Thanks!